config/migrations/2024/20241216090000-denderleeuw-duplicate-mandatarissen-iv.sparql

```diff
@@ -0,0 +1,59 @@
+      ( <http://data.lblod.info/id/mandatarissen/4ce5150c-8abd-47cd-a591-37023ed16bc7> <http://data.lblod.info/id/personen/079a77526135628ff778e9d1a1d5efe9e5073a0d46abc76e12820ed07de22461> ) # Ingrid - Temmerman
+      ( <http://data.lblod.info/id/mandatarissen/01c52041-5ad0-44b8-ace0-9e199140da14> <http://data.lblod.info/id/personen/d0718c59a8d56ef1f9e6ee82b5478e7f45b1a7241cf1d5212f2b47dd37c657ad> ) # Jan - De Nul
+      ( <http://data.lblod.info/id/mandatarissen/b113cdf1-fae3-497d-84a1-bbb6b8af9a1c> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo
+      ( <http://data.lblod.info/id/mandatarissen/300ab513-47c1-4bd0-8e9f-c0d16d76d289> <http://data.lblod.info/id/personen/5036d6fc-7689-4814-a76b-5d8f76e85b3b> ) # Jochen - Baert
+      ( <http://data.lblod.info/id/mandatarissen/fd06c501-848b-4fa0-a5b5-54c9d56567bc> <http://data.lblod.info/id/personen/f418356900e53a2c8f31b4bfed41106c31b04b971cf848bad715e2e8987dcd08> ) # Christel - De Bruecker
+      ( <http://data.lblod.info/id/mandatarissen/1ca7f25b-4197-44a1-aaad-9299f86b2a0a> <http://data.lblod.info/id/personen/7c0c0e90-0459-40a2-a6e8-aa7b110297f2> ) # Jonas - Cosyns
+      ( <http://data.lblod.info/id/mandatarissen/da47fc14-5d84-410b-8422-8649ecfc42b0> <http://data.lblod.info/id/personen/f65accfce0f8d29ffc80b171a7b4eb43dc556b1b58a0c2da38d9ac656c88024f> ) # Erna - Scheerlinck
+      ( <http://data.lblod.info/id/mandatarissen/b1d9be19-182e-47b9-91c9-8f2f19633809> <http://data.lblod.info/id/personen/bb1d87ff15870b5e8238c31e52d4915833c3880fe35415bfade055d2224b39cf> ) # Kristof - Slagmulder
+      # Samenstelling College van Burgemeester en Schepenen Denderleeuw
+      ( <http://data.lblod.info/id/mandatarissen/eff229ca-b021-49f5-97a7-36ff949f4f05> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo
+      ( <http://data.lblod.info/id/mandatarissen/53482b56-848b-4e8f-a18b-f4bd348d7b88> <http://data.lblod.info/id/personen/af09a92e521b3833dabc64d901cf62b66b4d653a2b381fe023a559b3bd754a5d> ) # Yves - De Smet
+      # Samenstelling Vast Bureau Denderleeuw
+      ( <http://data.lblod.info/id/mandatarissen/8e590c0d-8016-473f-b990-c03bd7197e5a> <http://data.lblod.info/id/personen/af09a92e521b3833dabc64d901cf62b66b4d653a2b381fe023a559b3bd754a5d> ) # Yves - De Smet
+      ( <http://data.lblod.info/id/mandatarissen/5ff1bac1-ba64-4a82-91a0-c8d9c72fa5f0> <http://data.lblod.info/id/personen/3b4692bf-e5aa-4a74-8c70-7f1fb18e663e> ) # Chancelvie - Okitokandjo # 2026!!
+    }
+
+      GRAPH <http://mu.semte.ch/graphs/organizations/856cbc8af3fbe7390d43420e249c288596e87fbc069c601751d5c5fcd52c543b/LoketLB-mandaatGebruiker> {
+      ?mandataris mandaat:isBestuurlijkeAliasVan ?persoon.
+      ?mandataris org:hasMembership ?membership.
+      ?mandataris ?p ?o.
+      OPTIONAL {
+        ?mandataris dcterms:modified ?modified.
```

Comment:
this optional doesn't matter. you're removing everything from the mandataris anyway

Reply:
Removed, indeed